lib/srv/alpnproxy/auth/auth_proxy_test.go

```diff
@@ -33,13 +34,15 @@ func TestDialLocalAuthServerNoServers(t *testing.T) {
 }
 
 func TestDialLocalAuthServerNoAvailableServers(t *testing.T) {
-	s := NewAuthProxyDialerService(nil /* reverseTunnelServer */, "clustername", []string{"0.0.0.0:3025"}, nil, nil)
-	_, err := s.dialLocalAuthServer(context.Background())
+	s := NewAuthProxyDialerService(nil /* reverseTunnelServer */, "clustername", []string{"203.0.113.1:3025"}, nil, nil)
```

Comment:
A quick comment about the reserved IP range might be nice here, since it otherwise looks valid-ish at first glance.